mlir/test/Transforms/remove-dead-values.mlir

```diff
@@ -687,3 +687,21 @@ func.func @op_block_have_dead_arg(%arg0: index, %arg1: index, %arg2: i1) {
   // CHECK-NEXT: return
   return
 }
+
+// -----
+
+
+// CHECK-LABEL: func @affine_loop_no_use_iv_has_side_effect_op
+func.func @affine_loop_no_use_iv_has_side_effect_op() {
```

Comment:
Can you add another test case specifically for the liveness analysis. E.g., in `test-liveness-analysis.mlir`.